docs/releases/5.1.txt

```diff
@@ -206,6 +206,9 @@ Tests
   :meth:`~django.test.SimpleTestCase.assertInHTML` assertions now add haystacks
   to assertion error messages.
 
+* Django test runner now supports ``--screenshots`` option to save screenshots
```

Comment:
In my mind, "The Django test runner" means `django.tests.runner.DiscoverRunner`. As far as I can tell, it's only `runtests.py` that supports the option. Since that's an internal utility, I wouldn't mention it in the release notes. If any of these features are meant to be public facing, they should be documented outside of `docs/internals` and the release note should link to it.

Reply:
We've documented new `runtests.py` options in this way in the past, e.g. e2de49ec2ea5827b36d0d4ec4f90b4ee0a003e93 or 8d010f39869f107820421631111417298d1c5bb9. I don't mind removing it :shrug:

Reply:
I've tried to clarify this in https://github.com/django/django/pull/17799.